### DISCUSSION TITLE
Since updating Aspire to preview 6 I am unable to deploy using azd

### DISCUSSION THREAD

**Discussion by Bigfellahull** [microsoft/aspire#3966] [General]
I've updated to Aspire preview 6 and the latest Azure CLI and am now unable to deploy to Azure container apps. It seems when provisioning, the actual project containers are not created.

I've been able to replicate this just following the guide (https://learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?tabs=macos) and using the Aspire starter template. If you run `azd provision --preview` you'll see the cache container app listed (assuming you enabled redis) but not the apiservice nor webfrontend.

I am running on MacOS 14.4.1. 

Can anyone confirm if they have this issue on Windows?

**Comment by davidfowl** [comment]
Windows works fine AFAIK. What version of azd are you running?

**Reply by Bigfellahull** [reply]
Version 1.8.1

**Reply by davidfowl** [reply]
Can you reproduce on a new preview 6 project?

**Reply by Bigfellahull** [reply]
I can. If I create a new project `dotnet new aspire-starter AspireSample` which uses the preview 6 packages,  and run `azd init` and then `azd provision` I see the same output. The resource group, container apps environment, container registry and log analytics workspace created. No project container apps.

Also, during `azd init` I am no longer asked to select which projects should be open to the internet.

**Reply by davidfowl** [reply]
> I can. If I create a new project dotnet new aspire-starter AspireSample which uses the preview 6 packages, and run azd init and then azd provision I see the same output. The resource group, container apps environment, container registry and log analytics workspace created. No project container apps.

In this new project, run azd infra synth. Let's see if something is busted with respect to the bicep/yaml generation.


> Also, during azd init I am no longer asked to select which projects should be open to the internet.

That's by design:

https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/preview-5?tabs=dotnet-cli#no-more-exposed-endpoint-selection-in-azd

**Reply by Bigfellahull** [reply]
Here is the Aspire Starter template after running `azd infra synth` [AspireSample.zip](https://github.com/dotnet/aspire/files/15132428/AspireSample.zip)

**Comment by davidfowl** [comment]
I see the container app manifests in your application and the bicep. Can you post the output of `azd up`?

**Comment by Bigfellahull** [comment]
Turns out I can deploy the starter project and the issue with my actual project is due to an unrelated error. I think I was getting confused with the starter project as I didn't see the output I expected from the provisioning preview. I am going to put this down to "It's a Friday afternoon".
